pkg/types/ccipocr3/plugin_commit_types.go

```diff
@@ -37,6 +55,24 @@ func NewMerkleRootChain(
 	}
 }
 
+// RMNECDSASignature is the ECDSA signature from a single RMN node
+// on the RMN "Report" structure that consists of:
```

Comment:
Is this the best place for this documentation? What happens if the report structure changes? Will be better if we link to the report struct directly or something.

Reply:
It's good to have documentation where possible.

Reply:
It's good when it's not something that will change regularly or there's no central place to link to (the original contract for example). I won't block the PR because of this but it can be more harmful if it diverges from the current documentation.

Reply:
Yeah, linking to the contract is fine, unfortunately its not finalized. But will put a link here anyway, and boyscout principle means that someone should update it to the latest relevant link at some point 😉

Reply:
Added link